liquidity-module/index.md

```diff
@@ -0,0 +1,291 @@
+---
+order: 0
+description: Use the Tendermint liquidity module to create pools, deposit to pools, and withdraw from pools with tokens sent using IBC.
+---
+
+# Understanding the Liquidity Module
+
+The liquidity module, known on the Cosmos Hub as Gravity DEX, enables users to create liquidity pools and swap tokens. 
+
+## Introduction
+
+When using your Cosmos SDK-based blockchains, you want to enable users to trade tokens. You can have multiple tokens on your blockchain or have tokens from external blockchains sent to your blocking using inter-blockchain communication protocol (IBC). The liquidity module allows users to use pools to trade those tokens on your blockchain. Each pool represents a token pair and allows the user to swap from one token to the other token.
+In this tutorial, you create your own blockchain, send tokens to another blockchain, create a pool, deposit to a pool, withdraw from a pool, and swap tokens.
+
+**Important** In the code examples throughout this tutorial, when you see username be sure to substitute your username. 
+
+**You will learn how to:**
+
+- Create a blockchain with Starport
+- Create your own token on your blockchain
+- Connect your blockchain to the testnet
+- Send your own token with IBC to the testnet
+- Create a pool with your token
+- Use the pool with your token
+
+## Prerequisites
+
+Before you start the tutorial, install the prerequisite software. 
+
+- [Install Starport](../starport/index.md) v0.16.2 <!-- link to the new tutorial file for this prereq to install Starport https://github.com/cosmos/sdk-tutorials/pull/694/commits/9a988d64408df16dad61412b7c542f6dd1fa4bee  -->
+
+    **Important** This tutorial uses [Starport](https://github.com/tendermint/starport) v0.16.2. The tutorial is based on this specific version of Starport and is not supported for other versions.
+
+- Install the Gravity DEX binary:
+
+    ```bash
+    git clone https://github.com/b-harvest/gravity-dex.git
+    cd gravity-dex
+    make install
+
+    gaiad version
+    ```
+
+    The output of `gaiad version` should print something like
+    
+    ```bash
+    gravity-dex-fa647b0fefe5508e9c975b3d4f095db2d3d20a13
+    ```
+
+## Create the Blockchain
```

Comment:
Have we removed the steps for deleting previous blockchain scaffolding ?

Can't seem to find it.